packages/phone-number-privacy/common/src/index.ts

```diff
@@ -7,3 +7,5 @@ export {
 export { toBool, toNum } from './utils/config-utils'
 export * from './utils/constants'
 export * from './utils/input-validation'
+export * from './test/utils'
+export * from './test/values'
```

Comment:
This isn't a big deal but it's polluting the 'source' namespace with 'test' stuff. Consider wrapping these under a Test object or something like that.